src/ocean/io/select/client/FileSystemEvent.d

```diff
@@ -153,6 +208,21 @@ class FileSystemEvent : ISelectClient
 
     /***********************************************************************
 
+        Replace the handle delegate
+
+        Params:
+            handler = event handler
+
+    ***********************************************************************/
+
+    public void setHandler ( DirHandler handler )
```

Comment:
It needs to be clearer that both a `Handler` _and_ a `DirHandler` can be set and that they will both be called.

Two questions on that topic, though:
1. Is there actually a use case for this or would it be better to have a union of handlers, instead?
2. Are the cases covered by `DirHandler` a superset of those covered by `Handler`? (i.e. now there's `DirHandler`, is there any reason to use `Handler`?) In this case, is it worth deprecating `Handler`?